incubator/kafka/values.yaml

```diff
@@ -134,6 +134,52 @@ persistence:
   ##
   # storageClass:
 
+## Metrics
+##
+metrics:
+
+  jmx:
+    enabled: false
+
+    # The image to use for the metrics collector
+    image: solsson/kafka-prometheus-jmx-exporter@sha256
+
+    # The image tag to use for the metrics collector
+    imageTag: a23062396cd5af1acdf76512632c20ea6be76885dfc20cd9ff40fb23846557e8
+
+    # The port to expose JMX metrics on
+    port: 5555
+
+    # Rules to apply to the Kafka JMX Exporter
+    kafkaConfig:
+      lowercaseOutputName: true
+      jmxUrl: service:jmx:rmi:///jndi/rmi://127.0.0.1:5555/jmxrmi
+      ssl: false
+      whitelistObjectNames: ["kafka.server:*", "kafka.controller:*", "java.lang:*"]
+      # rules:
```

Comment:
Perhaps these should just be set as defaults, what do you think (eg not commented)?

Reply:
My reasoning for leaving them commented was because including them excludes a lot of metrics; I'm personally of the opinion that the default should include everything, allowing a user to exclude stuff if necessary, rather than hide metrics by default, requiring a user to figure out why they are missing and how to expose them. Typically articles/help guides about monitoring in general will reference using specific metrics, and it can be confusing while going through those if the metrics that _should_ be there happened to be hidden by default.

I left them in and commented mostly as a reference in case someone did want to write their own rules, but if the choice is between uncommenting, or removing entirely, I would err on the side of "just remove them"

Reply:
Seems reasonable to me 👍